config.json

```diff
@@ -1441,6 +1441,21 @@
       "difficulty": 0,
       "topics": null,
       "deprecated": true
+    },
+    {
+      "slug": "connect",
+      "uuid": "1a714bd0-ed15-4b19-8f54-19c6e269763b",
+      "core": false,
+      "unlocked_by": "grade-school",
```

Comment:
Exercises are sorted by difficulty level. Please put it the the correct difficulty level.

Also, please correct the error "The exercise 'connect' is being unlocked by a non-core exercise. Non-core exercises can only be unlocked by core exercises."